app/styles/default.js

```diff
@@ -7,4 +7,10 @@ export default StyleSheet.create({
     alignItems: 'center',
     backgroundColor: '#F5FCFF',
   },
+  matrixInner: {
+    flex: 1,
+  },
+  matrixOuter: {
+    
```

Comment:
**<a href='http://eslint.org/docs/rules/no-trailing-spaces'>no-trailing-spaces</a>**: Trailing spaces not allowed.